docs_src/pluggables/manual.py

```diff
@@ -38,6 +34,6 @@ async def home(request: Request) -> JSONResponse:
 
 
 app = Esmerald(routes=[Gateway(handler=home)])
-
-extension = MyExtension(app=app)
-extension.extend()
+app.add_pluggable("my-extension", MyExtension)
+# or
```

Comment:
Give an extra return here please and 

```python
# Or another option can be

app.add_extension("my-extension", MyExtension)
```

As not a comment. Its documentation we are talking about.

```